roles/openshift_hosted/defaults/main.yml

```diff
@@ -1,14 +1,33 @@
 ---
+##########
+# Common #
+##########
+openshift_hosted_infra_selector: "region=infra"
+r_openshift_hosted_use_calico_default: "{{ openshift_use_calico | default(False) }}"
+r_openshift_hosted_use_calico: "{{ r_openshift_hosted_use_calico_default }}"
+
+openshift_default_projects:
+  default:
+    default_node_selector: ''
+  logging:
+    default_node_selector: ''
+  openshift-infra:
+    default_node_selector: ''
+
+# openshift_additional_projects shares the same format as openshift_default_projects
+openshift_additional_projects: {}
+
+openshift_config_base: "/etc/origin"
+openshift_master_config_dir: "{{ openshift.common.config_base }}/master"
```

Comment:
If openshift.common.config_base is not set for some reason consider:

```
"{{ openshift.common.config_base | default(openshift_config_base) }}/master"
```

It looks like you defined the variable on the previous line.  Hopefully in the future we won't need all of the facts.

Reply:
I agree with this sentiment.  I originally made a lot of changes along those lines, and this patch set got lost in the weeds.  I'm trying to scale back on the refactors, just moving some things around here.